src/Compilers/Core/Rebuild/MetadataReferenceInfo.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.CodeAnalysis.Rebuild
         public readonly string Name;
         public readonly FileInfo FileInfo;
         public readonly Guid Mvid;
-        public readonly ImmutableArray<string> ExternAliases;
+        public readonly string? ExternAlias;
```

Comment:
why did we delay splitting the alias names?

Reply:
My guess is that the original implementation attempted to represent extern aliases in the same way that MSBuild does. Essentially there is one reference with a collection of aliases. In reality though the compiler ends up representing it more as a 1-1 mapping by the time you get to the command line. This change better reflects the compiler representation and is more suited to reubild scenarios.